matsim/src/main/java/org/matsim/core/config/consistency/VspConfigConsistencyCheckerImpl.java

```diff
@@ -242,7 +243,7 @@ public void checkConsistency(Config config) {
 		// === plansCalcRoute:
 		
 		// added feb'16
-		if ( !config.plansCalcRoute().isInsertingAccessEgressWalk() ) {
+		if ( config.plansCalcRoute().getAccessEgressWalkType().equals(AccessEgressWalkType.None) ) {
```

Comment:
maybe you could correct the warning as well, since 'true' and 'false' can't be set anymore